addons/docs/src/frameworks/web-components/custom-elements.ts

```diff
@@ -39,15 +50,18 @@ function mapData(data: TagItem[], category: string) {
   return (
     data &&
     data.reduce((acc, item) => {
-      const type = category === 'properties' ? { name: item.type } : { name: 'void' };
+      if (item?.kind === 'method') return acc;
```

Comment:
Can `item` not be defined here? or is the `?` an extra security just to be sure?

Reply:
In the `'experimental'` version of the schema, `item` wont have a `kind` here, but in the `'1.0.0'` version of the schema, it does, and we want to be sure its not a method